src/explanation/shap_wrapper.py

```diff
@@ -21,7 +37,7 @@ def explain(shap_exp: Explanation, training_df, test_df, explanation_target, pre
     model = model[0]
     prefix_int = int(prefix_target.strip('/').split('_')[1])-1
 
-    explainer = _init_explainer(model)
+    explainer = _init_explainer(model, training_df)
```

Comment:
The explanation object contains a reference to the predictive_model, please pass to the function you customised the 'prediction_method' string contained in the PredictiveModel, otherwise your code will never be executed.